.github/workflows/do-release.yml

```diff
@@ -0,0 +1,158 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+    - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+name: Create draft release
+
+jobs:
+  create_release:
+    name: Create the Github Release
+    runs-on: ubuntu-latest
+    needs: [release-osx, release-ubuntu-16-04]
+    steps:
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ github.ref }}
+          release_name: Release ${{ github.ref }}
```

Comment:
Does this match the release name to be the semver tag?

Reply:
I think it will make it `release-0.4.10` etc